dbt/include/sqlserver/macros/materializations/snapshot/strategies.sql

```diff
@@ -4,7 +4,7 @@
     {% endfor %}), 2)
 {% endmacro %}
 
-{% macro sqlserver__snapshot_check_strategy(node, snapshotted_rel, current_rel, config, target_exists) %}
+{% macro snapshot_check_strategy(node, snapshotted_rel, current_rel, config, target_exists) %}
```

Comment:
why did this go back to `snapshot_check_strategy()`? whereas in  #29  adding back the `sqlserver__` prefix? I'm confused...

Reply:
Hi, 

I am not 100 sure why it sometimes needs to be named sqlserver__ and sometimes not. But when trying to develop the adapter for 0.16 I got the error that there already was a macro named i.e. `make_relation` and that there could not be two with the same name. As per instructions I renamed all to sqlserver__. When developing for 0.18 the `sqlserver__snapshot_check_strategy` was not picked up and I got the instruction to just name it `snapshot_check_strategy`. See https://getdbt.slack.com/archives/C50NEBJGG/p1598361666015200 for more info.